.lintr

```diff
@@ -31,7 +31,7 @@ linters: linters_with_defaults(
     sort_linter(),
     sprintf_linter(),
     strings_as_factors_linter(),
-    undesirable_function_linter(c(Sys.setenv = NA_character_, mapply = NA_character_)),
+    undesirable_function_linter(c(Sys.setenv = NA_character_, mapply = NA_character_, structure = NA_character_)),
```

Comment:
Didn't we clean this up?

Reply:
Not sure I follow...

Reply:
NVM I had thought we were using all default undesirables.